docs/scripts/generate-sitemap-robotstxt.mjs

```diff
@@ -24,33 +24,68 @@ async function generateSitemap() {
     '!src/pages/404.page.tsx',
   ]);
 
-  const pages = pagesWithParam.flatMap((p) =>
-    ['react', 'angular', 'vue', 'flutter'].map((framework) =>
-      p.replace('[platform]', framework)
-    )
-  );
+  const pages = pagesWithParam.flatMap((p) => {
+    p = p
+      .replace('src/pages', '')
+      .replace('data', '')
+      .replace('.js', '')
+      .replace('.page.mdx', '')
+      .replace('.page.tsx', '')
+      .replace('/index', '');
```

Comment:
I like the clarity of using `replace` for each string. Part of me wonders if a single regex would be more slightly more efficient?

`const regex = /src\/pages|data|.js|.page.mdx|.page.tsx|\/index/`
`p = p.replace(regex, '')`

Reply:
Good idea! I'm also thinking since our files are changing, it might be good to choose "clear" in this case so that it's easier to update the script?